api/docker/web/health-check.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
Is it not possible to just use docker health checks and curl for this, or is it because we're doing multi-endpoint health checking and you're trying to combine outputs?

Reply:
The second option. This is just moving this health-check over to the api side as it was pulling from the client side so this has been running for a long time as is. I can explain the full reasoning behind this healthcheck in a convo after standup if you like as it has a bit of history.